.github/workflows/test.yml

```
@@ -63,6 +65,11 @@ jobs:
         with:
           use-github-cache: false
 
+      - name: Set CUDA 12.9 env vars
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2016:info:1:6: Expressions don't expand in single quotes, use double quotes for that

(shellcheck)